README.md

```diff
@@ -1,6 +1,81 @@
 # Beschreibung Patterns-and-Frameworks
 Semesterprojekt zur Entwicklung eines auf Java/JavaFX basierenden Computerspiels.
 
+# Spielablauf
+
+### Startseite und Highscore
+Auf der Startseite befindet sich eine Highscore Tabelle mit allen Teams und den jeweiligen Leveln und der 
+erreichten Punktzahl.
+Über die Startseite kann sich ein Spieler registrieren oder einloggen, sein Profil ändern, ein Team anlegen 
+oder einem Team beitreten oder ein Spiel starten.
```

Comment:
Ist ein Spieler eingeloggt und ist er in einem aktiven Team registriert, so werden im Top Panel der Startseite die aktuellen Teamdaten geladen.